spec/features/admin_views_one_family_spec.rb

```diff
@@ -0,0 +1,12 @@
+require "rails_helper"
+
+feature "Admin can see a family" do
+  scenario "successfully" do
+    family = create(:family, last_name: "Yellow")
+
+    visit "/families/#{family.id}"
```

Comment:
Could you use `family_path(family)` here?

Reply:
👍